index.html

```diff
@@ -106,6 +106,68 @@ <h3>
       <h2>
         Examples
       </h2>
+      <h3>
+        Loading and parsing
+      </h3>
+      <p>
+        A model element whose source path resolves to a valid resource
+        dispatch a `load` event upon the successful fetching and 
+        presentation of the resource data. If the asset fails to load,
+        for instance due to an invalid URL, a failure to fetch,
+        or a failure in the source format's parsing process, the user agent dispatches an 
+        `error` event.
+      </p>
+      <aside class="example" title="Error handling">
+      <pre class="js">
+      const model = document.querySelector("model");
+      model.addEventListener("error", () => {
+        // Handle error 
+      });
+      </pre>
+      </aside>
+      <h3>
+        Processing model contents
+      </h3>
+      <p data-cite="infra geometry-1" >
+        A 3D asset file contains positional data for the surfaces and objects
+        within it. Following the successful processing of a model resource, the
+        {{Model}}'s {{Model/boundingBox}} attribute references a
+        [[\DOMBoundingBoxReadOnly]] internal slot, which 
+        is a nullable [=map=] of whose values are {{DOMPoint}}s. 
+        The slot contains four entries: 
+      </p>
+      <dl data-cite="infra geometry-1">
+         <dt>"min"</dt>
+         <dd>A {{DOMPoint}} representing the minimum dimension, in world
+          space, that the contents of a model occupies.</dd>
+         <dt>"max"</dt>
+         <dd>A {{DOMPoint}} representing the maximum dimension, in world
+          space, that the contents of a model occupies.</dd> 
+         <dt>"center"</dt>
+         <dd>A {{DOMPoint}} representing the mean of the min and max 
+          dimensions, in world space, of the bounding box that the contents of
+          a model occupies.</dd>
+         <dt>"radius"</dt> 
+         <dd>A floating-point number representing the radius of effective 
+          bounding sphere from the center of the box. 
+        </dd>
+      </dl>
+      <p>
+       If a model does not contain a valid resource, the [[\DOMBoundingBoxReadOnly]] internal slot is null.
+      </p>
+      </p>
+      <figure>
+      <img alt="" src=
+      "images/bounding-box.svg">
+      <figcaption>Visual aid demonstrating the extent of an object's bounding box</figcaption>
+      </figure>
+      <aside class="note">
+        The bounding box and sphere is often computed based on the visible object
+        hierarchy on the first animation frame (if an animation is present),
+        and according only to the geometry information directly specified.
```

Comment:
nit: "specified" by who? (can we change this to active voice?)

Reply:
updated to reflect that rendering contexts generally compute it this way - it may need to be called out as non-normative or otherwise an observation of the state of the art.